src/global-styles.js

```diff
@@ -1,10 +1,10 @@
-import { injectGlobal } from 'styled-components';
+import { createGlobalStyle } from 'styled-components';

```

Comment:
Hm, so with the new API, I think this should become a component, living in `src/components/GlobalStyles/GlobalStyles.js` (with an `index.js` that links to it, see existing components for examples).